cvat-ui/src/components/analytics-report/analytics-report-page.tsx

```diff
@@ -0,0 +1,204 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import './styles.scss';
+
+import React, { useState, useEffect, useCallback } from 'react';
+import { shallowEqual, useSelector } from 'react-redux';
+import notification from 'antd/lib/notification';
+import { useLocation, useParams } from 'react-router';
+import { Row, Col } from 'antd/lib/grid';
+
+import {
+    Project, Task, Job, getCore, MembershipRole, InstanceType,
+} from 'cvat-core-wrapper';
+import { CombinedState } from 'reducers';
+import GoBackButton from 'components/common/go-back-button';
+import CVATLoadingSpinner from 'components/common/loading-spinner';
+import AnalyticsReportContent from './analytics-report-content';
+import AnalyticsPageHeader from './analytics-page-header';
+
+const core = getCore();
+
+function useInstanceType(): InstanceType {
```

Comment:
I would suggest to move those 2 funcitons into utils as they will be useful for quality page too

Reply:
Moved